kostyor/tests/unit/rest_api/test.py

```diff
@@ -339,3 +340,61 @@ def test_discover_cluster_discovery_failed_error_response(self,
                                               auth_url='http://9.9.9.9')
         fake_session.assert_called_once_with(auth=password_mock)
         fake_discover.assert_called_once_with()
+
+    @mock.patch.object(db_api, 'get_cluster', mock.Mock())
+    @mock.patch.object(db_api, 'get_hosts_by_cluster', mock.Mock())
+    def test_host_list_cluster_exists_success(self):
+        fake_hosts = ['hostname_1', 'hostname_2']
```

Comment:
Despite you deal with a mock and test is passed, I think it would be better to provide values closed to real ones.  Otherwise, there might be holes in our tests.

According to `get_hosts_by_cluster` implementation, it returns a list of objects, not strings. So could you please prepare mocks according to `Host` schema?